libscript/src/objc.lcb

```diff
@@ -610,4 +610,89 @@ public handler CreateObjcInformalDelegateWithContext(in pProtocol as List, in pH
     end unsafe
 end handler    
 
+/* Block handling */
+
+type LibraryHandle is Pointer
+
+private variable mNSConcreteGlobalBlock as optional Pointer
+
+constant kRTLD_LAZY is 1
+private foreign handler OpenLibrary(in pPath as ZStringUTF8, in pFlag as CSInt) returns optional LibraryHandle binds to "dlopen"
```

Comment:
Break the foreign handler definitions as we have above - i.e. continuation to newline before the `binds`